web/docs/Backend Starters/Decoupled WordPress/configuring-build-hooks.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 'configuring-build-hooks'
+title: 'Configuring Build Hooks'
+slug: '/backend-starters/decoupled-wordpress/configuring-build-hooks'
+sidebar_position: 2
```

Comment:
2 is currently taken, i think this one is 5. We need a better system for this though, it's not fun to manage "by hand".